README.md

```diff
@@ -135,17 +135,17 @@ And for `print_number`, we'll use `processopedia -t print_number -d`:
 
 The output of these commands tells us enough about each process to construct a Sprocket ".pipe" file that defines a processing pipeline.  In particular we'll need to know how to configure each process (the "Configuration") and how they can be hooked together (the input and output "Ports").
 
-KWIVER comes with a sample [kwiver/pipeline_configs/number_flow.pipe](kwiver/pipeline_configs/number_flow.pipe) file that configures and connects the pipeline so that the `numbers` process will generate a set of integers from 1 to 99 and the `print_number` process will write those to a file called `numbers.txt`.  Of particular interest is the section at the end of the file that actually "hooks up" the pipeline.
+KWIVER comes with a sample [sprokit/pipelines/number_flow.pipe](sprokit/pipelines/number_flow.pipe) file that configures and connects the pipeline so that the `numbers` process will generate a set of integers from 1 to 99 and the `print_number` process will write those to a file called `numbers.txt`.  Of particular interest is the section at the end of the file that actually "hooks up" the pipeline.
 
 To run the pipeline, we'll use the Sprokit `pipeline_runner` command:
 
-	pipeline_runner -p </path/to/kwiver/source>/kwiver/pipeline_configs>/number_flow.pipe
+	pipeline_runner -p </path/to/kwiver/source>/sprokit/pipelines/number_flow.pipe
 
 After the pipeline completes, you should find a file, `numbers.txt`, in your working directory.
 
 ### Python Processes
 
-One KWIVER's great strengths (as provided by Sprokit) is the ability to create hybrid pipelines which combine C++ and Python processes in the same pipeline.  This greatly facilitates prototyping complex processing pipelines.  To test this out we'll still use the `numbers` process, but we'll use a Python version of the `print_number` process called `kw_print_number_process` the code for which can be seen in [kwiver/processes/kw_print_number_process.py](kwiver/processes/kw_print_number_process.py).    As usual, we can lean about this process with a `processopedia` command: `processopedia -t kw_print_number_process -d`:
+One KWIVER's great strengths (as provided by Sprokit) is the ability to create hybrid pipelines which combine C++ and Python processes in the same pipeline.  This greatly facilitates prototyping complex processing pipelines.  To test this out we'll still use the `numbers` process, but we'll use a Python version of the `print_number` process called `kw_print_number_process` the code for which can be seen in [sprokit/processes/python/kw_print_number_process.py](sprokit/processes/python/kw_print_number_process.py).    As usual, we can lean about this process with a `processopedia` command: `processopedia -t kw_print_number_process -d`:
```

Comment:
Not your fault, but might as well fix it while we're in here: "One [of] KWIVER's..."